app/models/answer_group.rb

```diff
@@ -4,4 +4,5 @@
 # An AnswerGroups's parent is an AnswerGroupSet or an AnswerGroup.
 # Its children can be Answers, AnswerSets, AnswerGroups, or AnswerGroupSets.
 class AnswerGroup < ResponseNode
+  belongs_to :questioning, class_name: "QingGroup"
```

Comment:
Name this association `qing_group` instead? We eventually need to refactor the db so that the col name is `form_item_id` instead of `questioning_id` anyway...